addons/account/models/account_journal_dashboard.py

```diff
@@ -493,6 +493,8 @@ def open_payments_action(self, payment_type, mode='tree'):
             action_ref = 'account.action_account_payments'
         [action] = self.env.ref(action_ref).read()
         action['context'] = dict(ast.literal_eval(action.get('context')), default_journal_id=self.id, search_default_journal_id=self.id)
+        if payment_type == 'transfer':
```

Comment:
what's that?

Reply:
Called by the dashboard here:
https://github.com/odoo/odoo/blob/master/addons/account/views/account_journal_dashboard_view.xml#L71

This will not be set on an account.payment :)